Kornilich xenia

ksyukornilich opened the pull request.

ksyukornilich committed:
change readme

ksyukornilich committed:
Create lol2

ksyukornilich committed:
Add files via upload

ksyukornilich committed:
Задание №1

ksyukornilich committed:
Задание 2

ksyukornilich committed:
Задание 3

Nastasia8 commented on vector.cpp:
```diff
@@ -0,0 +1,52 @@
+           cout << "Vc = " << Vc << endl;    // векторное произведение (1)
+           
+    return 0;
+  }
```
Конечно, не хватает отдельных функций, но так тоже ок)

Nastasia8 commented on lol1.cpp:
```diff
@@ -0,0 +1,48 @@
+   cin >> count;
+   
+
+   switch (count)
```
а если не 1 и не 2?))

Nastasia8 replied:
Т.е. если пользователь введет что-то другое 29, 13, и т.д.

ksyukornilich commented:
А как понять не 1 и не 2? Что вы имеете ввиду?)
--
Отправлено из Mail.ru для Android воскресенье, 24 октября 2021г., 21:11 +03:00 от Nastasia8  ***@***.*** :

***@***.*** commented on this pull request.
>----------------------------------------------------------------------
>In  vector.cpp :
> +      b=sqrt(pow(x2,2)+pow(y2,2)+pow(z2,2)); 
>+           cout << "a = " << a << endl; 
>+           cout << "b = " << b << endl; 
>+
>+      Sc=x1*x2+y1*y2+z1*z2;
>+           cout << "Sc = " << Sc << endl;    // скалярное произведение (1)
>+
>+      cosine=Sc/(a*b);
>+
>+      sine=sqrt(1-pow(cosine,2));
>+
>+      Vc=a*b*sine;
>+           cout << "Vc = " << Vc << endl;    // векторное произведение (1)
>+           
>+    return 0;
>+  }
>
>Конечно, не хватает отдельных функций, но так тоже ок)
>----------------------------------------------------------------------
>In  lol1.cpp :
> +   float A[i]={0.08,0.26,0.35,0.41,0.53};
>+   int a=2, b=3;
>+
>+   float y=0;
>+   float g=0;
>+   
>+   float Xn=0.11;
>+   float Xk=0.36;
>+   float dX=0.05;
>+
>+   int count;
>+   cout<<"write 1 or 2";
>+   cin >> count;
>+   
>+
>+   switch (count)
>
>а если не 1 и не 2?))
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  unsubscribe .
>Triage notifications on the go with GitHub Mobile for  iOS or  Android .

ksyukornilich committed:
Задание 4 (1)

ksyukornilich committed:
Задание 4 (5)

ksyukornilich committed:
Задание 4 (4)

ksyukornilich committed:
Задание 4 (1)

ksyukornilich committed:
Задание 4 (2)

ksyukornilich committed:
Задание 4 (3)

ksyukornilich committed:
Задание 4 (6)

Nastasia8 commented on laba 4/schet.cpp:
```diff
@@ -0,0 +1,69 @@
+         cout << "pw = " << pw << endl;
+        break; 
+      }
+  }
```
ветки default не хватает

Nastasia8 commented on laba 4/discrim(5).cpp:
```diff
@@ -0,0 +1,44 @@
+
+int D(float x, float y, float z, float f1, float f2, float ds)
+{ 
+    return pow(y,2)-4*x*z;
```
Зачем переменные f1,  f2, ds, если они не используются?

ksyukornilich replied:
долго правила код, забыла убрать(

Nastasia8 commented on laba 4/discrim(5).cpp:
```diff
@@ -0,0 +1,44 @@
+                {
+                    float f1;
+                     f1=(-y+sqrt(ds))/(2*x);
+                    float f2;
```
Вы же уже в 16 и 17 строчках объявили переменные, зачем повторять здесь?

Nastasia8 commented on laba 4/chetn(4).cpp:
```diff
@@ -0,0 +1,31 @@
+      {
+        S=S+n;
+      }
+        else S=S+0;
```
Лишнее условие в 12 строке, все и так будет работать

Nastasia8 commented on laba 4/chetn(4).cpp:
```diff
@@ -0,0 +1,31 @@
+
+int main()
+{   
+    int S=0;
```
И сумму лучше было объявить внутри функции SM()

Nastasia8 commented on laba 4/6zn(6).cpp:
```diff
@@ -0,0 +1,45 @@
+
+    cin >> A;
+
+    zn(b0, Pr, A, b, b1, b2, b3, b4, b5);
```
Зачем нужно передавать столько переменных (у вас там на данный момент "мусор"), достаточно просто передать A)

Nastasia8 commented on laba 4/6zn(6).cpp:
```diff
@@ -0,0 +1,45 @@
+    b0=b0/10;
+
+    b4=b0%10;
+    b0=b0/10;
```
одинаковые действия явно напрашивается цикл)

ksyukornilich replied:
Знаю! Но через цикл не вышло(

ksyukornilich committed:
Задание 4 (7)

ksyukornilich committed:
Задание 4 (7)

ksyukornilich committed:
Задание 5 (1)

ksyukornilich committed:
Задание 5 (2)

ksyukornilich committed:
Задание 4 (8)

ksyukornilich committed:
Задание 4 (8) исправила (доделала поиск минимального

ksyukornilich committed:
Задание 5 (3-4)

Третий и четвертый пункт в одном коде

ksyukornilich committed:
Задание 5 (5)

Стремные названия файлов, пишу для себя, чтобы не забыть о чем была лабораторная))

ksyukornilich committed:
Задание 6 (1)

ksyukornilich committed:
Задание 6 (2)

ksyukornilich committed:
Задание 6 (3-4)

ksyukornilich committed:
Задание 6 (*5)

Nastasia8 commented on laba 4/risunok(7).cpp:
```diff
@@ -0,0 +1,43 @@
+    cin >> sim;
+
+    RIS(height, width, sim);
+    return 0;
```
точно отрисовывает?)

Nastasia8 commented on laba 5/dinmassiv(3-4).cpp:
```diff
@@ -0,0 +1,51 @@
+
+    delete [] a; // удаление динамического массива и освобождение его памяти
+    return 0;
+}
```
Что здесь пошло не так)))
Присмотритесь к коду)

Nastasia8 commented on laba 6/sisschis(2).cpp:
```diff
@@ -0,0 +1,42 @@
+        cout << "white new value of 'nom'" << endl;
+    }
+    return 0;
+}
```
20 в 2с.с. чему будет равно?) посмотрите, перевод числа 325)
Проверьте повнимательнее)

Nastasia8 commented on laba 6/NOD-NOK(3-4).cpp:
```diff
@@ -0,0 +1,58 @@
+    cout << "NOK = " << NOK( x, y, mod, mod1, nok1) << endl;
+   
+    return 0;
+}
```
А для таких примеров NOD(48, 36) или NOD(30, 40)  будет верно отображаться?)
